### ISSUE TITLE
Can't install necessary packages  error: 399113 cnchi problem again and again

### ISSUE THREAD

**Issue by pozitronij** [Antergos/Cnchi#685]
### Problem:
These is the same problem that is draging around for a while. Only Gnome 3 install is working. I triend XFCE multiple times, withought selecting any aditional optiopns, selecting diferent mirrors, and using latest Antergos ISO 30.12.2016. The problem remains. I tried also and MATE, still error code 399113

**Comment by lots0logs** [comment]
This issue has been linked to an error in Bugsnag
[**LogERROR** in **pacman/pac.py:157**](https://app.bugsnag.com/antergos/cnchi/errors/579c054175079c5ec92817f7?event_id=586860401966e611000feecf)

**Comment by lots0logs** [comment]
@pozitronij I've located two errors that occurred during the install attempt for which you provided us with the ID (39913). 

The first one is caused by the grub theme package failing to install. This commonly occurs when retying the installation after a failed attempt without rebooting your system first. Please let me know if that's what happened in this case. 

The second one I'm not sure about. We will have to look into it. I've performed three test installs without any issues just yesterday so my gut tells me your issue is likely related to mirrors. Please let us know if you have any more failed attempts and be sure to provide the install ids. Also, always reboot prior to trying again. Cheers!

**Comment by pozitronij** [comment]
Yes. I did restarted the system before each install atempt. Also i tried KDE , it fails, on the same way, but rerror is different in these case, i forgot what it was but it started with 3xxxxxx. I'll try once more.

**Comment by nabei** [comment]
have the same issue 
http://pastebin.com/8ZRpBgwZ

**Comment by karasu** [comment]
@lots0logs You can ignore the error in the hardware module. I'll fix it asap, but it's not fatal.

**Comment by lots0logs** [comment]
@nabei Your issue is not the same as is being reported on this github issue. Why are you cloning the cnchi repo to run the installer? Running Cnchi from the antergos user home directory will cause issues b/c it will end up sourcing files from both directories (the properly installed copy and the copy you cloned).

You shouldnt have to mess with any of that. It should auto update when you boot the ISO. If you are having trouble with that please open an issue for it here on github. Thanks :smiley:
